bridge/modules/bridge/module.yaml

```diff
@@ -19,12 +19,13 @@ packages:
   manager: microdnf
   content_sets:
     x86_64:
-      - rhel-7-server-rpms
+      - rhel-8-for-x86_64-baseos-rpms
   install:
     - unzip
     - tar
     - rsync
     - shadow-utils # groupadd
+    - gzip
```

Comment:
why this addition?

Reply:
`gzip` is needed by tar to handle `tar.gz` files. It was shipped automatically with `ubi7-minimal` image but not with the new `ubi8-minimal` image so we have to install it explicitly

Reply:
Why did they remove it!?? Anyway, thanks!